utils/_context/_scenarios/core.py

```diff
@@ -11,6 +11,7 @@
 class ScenarioGroup(Enum):
     ALL = "all"
     APPSEC = "appsec"
+    APPSEC_RASP = "appsec_rasp"
```

Comment:
RASP meaning Runtime Application Security Protection (or Platform, can't remember?) AppSec RASP seams redundant, WDYT ?

Reply:
"Runtime Application Self-Protection" ;-)

> AppSec RASP seams redundant

yes, could be. I set appsec prefix because I think that could help to find it, and also rasp if part of our appsec security catalog, where we have waf, api security, rasp...